example/android/build.gradle

```diff
@@ -3,16 +3,21 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
-        classpath 'com.google.gms:google-services:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.google.gms:google-services:3.2.1'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
+ext {
+    googlePlayServicesVersion   = "15.0.1"
+}
+
 allprojects {
     repositories {
         mavenLocal()
```

Comment:
I had to swap `jcenter()` and `maven { url "https://maven.google.com" }` lines to install dependencies. I got this error

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':react-native-google-signin:debugCompileClasspath'.
> Could not find common.jar (android.arch.core:common:1.0.0).
  Searched in the following locations:
      https://jcenter.bintray.com/android/arch/core/common/1.0.0/common-1.0.0.jar
```

Reply:
I had a similar problem with my other app that manifested itself after the bintray outage so I do thing it is related, but I will fix this.

Reply:
fixed